### PR TITLE
styles(tier-deductible-selector): add hover state

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Added hover state for tiers/deductible selector.

https://github.com/user-attachments/assets/08d7505b-bbd9-4c6e-9110-8f9dae3e0b6e

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
* **#4901** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4901?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @guilhermespopolin and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by alebedev — APPROVED** [review]
Nice!

File: apps/store/src/features/priceCalculator/PurchaseFormV2/OfferPresenterV2/CardRadioGroup/CardRadioGroup.tsx [hunk 1]
```diff
@@ -20,12 +20,24 @@ export function Item({
...
     unselected: {
       backgroundColor: tokens.colors.opaque1,
+      transition: {
+        duration: framerTransitions.defaultDuration,
+        ...framerTransitions.easeInOutCubic,
+      },
+    },
+    hover: {
+      backgroundColor: tokens.colors.opaque2,
+      transition: { duration: 0.1, delay: 0, ...framerTransitions.easeOutCubic },
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/PurchaseFormV2/OfferPresenterV2/CardRadioGroup/CardRadioGroup.tsx:40]: Do we need delay? I assume 0 is the default

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Oct 28, 5:03 AM EDT**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4901).

**Pull request merged**